.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing to pyccel
+
+Firstly, thanks for your interest in contributing code to pyccel!
+
+We welcome any and all contributions.
+
+There are many ways to help with the pyccel project which are more or less involved.
+The following is a summary of how you can do this.
+
+## Asking questions
+
+For simple questions (e.g. installation problems), we encourage you to use the [discussions tab](https://github.com/pyccel/pyccel/discussions/categories/q-a).
+
+## Requesting features
+
+To request a new feature, please use the [discussions tab](https://github.com/pyccel/pyccel/discussions/categories/ideas).
+This allows us to discuss the best implementation in the target languages before creating an issue to work on.
+
+## Reporting issues
+
+When reporting issues please include as much detail as possible about your
+operating system, and python version. Whenever possible, please
+also include a brief, self-contained code example that demonstrates the problem.
+
+## Answering questions
+
+Questions are sometimes asked in the [discussions tab](https://github.com/pyccel/pyccel/discussions/categories/q-a) or in issues.
+Any help answering these questions is appreciated as it helps our users find solutions as fast as possible.
```

Comment:
Maybe we can move this section to just after the "Asking questions" section?

Reply:
Fixed 77e5ae45

My aim was to order by ease so the further down the list you go, the more knowledge you need. But I agree that it makes sense to have asking/answering questions together